src/libs/Navigation/Navigation.ts

```diff
@@ -428,6 +428,13 @@ function getTopMostCentralPaneRouteFromRootState() {
     return getTopmostCentralPaneRoute(navigationRef.getRootState() as State<RootStackParamList>);
 }
 
+function getReportRouteByID(reportID?: string) {
```

Comment:
Ok sorry, one more dumb question. Why isn't the report route just `/r/<reportID>` ?

Reply:
Ok gotcha. Sounds good.